convex/schema.ts

```diff
@@ -22,6 +23,7 @@ const schema = defineEntSchema({
 
   messages: defineEnt({})
     .field("content", v.string())
+    .field("type", v.string(), { default: "message" })
```

Comment:
This is the type of a message. 
I need a difference between the normal messages and the request messages